tests/unit/ViolationUtilsTest.js

```diff
@@ -0,0 +1,505 @@
+import Onyx from 'react-native-onyx';
+import ViolationsUtils from '@libs/Violations/ViolationsUtils';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+describe('getViolationsOnyxData', () => {
+    let transaction;
+    let transactionViolations;
+    let policyRequiresTags;
+    let policyTags;
+    let policyRequiresCategories;
+    let policyCategories;
+
+    beforeEach(() => {
+        transaction = {category: 'Food', tag: 'Lunch', transactionID: '123'};
+        transactionViolations = [
```

Comment:
Would it be better to define this as an empty array, and that way the tests are more obvious about whether ViolationsUtils.getViolationsOnyxData is adding something, removing something, or leaving it untouched. 

Right now the tests read more as "violations haven't changed", which is good, but a bit less clear imo

Reply:
This makes sense. I updated the test so the base case is empty, and added tests to make sure the new violations were added to the existing list if one exists. I think it's a lot clearer now.